### PR TITLE
OE-835: Move email to constants

### PR DESCRIPTION
**Pull request by oliviawongnyc**
- Moves `CONTACT_EMAIL` to `constants` package for use in both OEW and OEH
- Ticket [OE-835](https://jira.nypl.org/browse/OE-835)

### CODE REVIEW

**Review by samanthaandrews — APPROVED** [review]
Code looks good. Let's have QA run through this on the web and hybrid apps just to be sure functionality is working as expected.

File: apps/oeh/src/components/Settings/ReportAProblem.tsx [hunk 1]
```diff
@@ -15,7 +16,7 @@ const ReportAProblem: React.FC = () => {
         <IonItem lines="none">
           <IonIcon icon={helpOutline} slot="start" />
           <IonLabel>Report a problem</IonLabel>
-          <a href="mailto:info@openebooks.org">
+          <a href={CONTACT_EMAIL}>
```
Comment (Toxiapo) [inline apps/oeh/src/components/Settings/ReportAProblem.tsx:19]: This needs the `mailto` keyword. : )

**Review by Toxiapo — CHANGES_REQUESTED** [review]
Just that one comment.

We can also move `DEV_CONTACT_EMAIL` to the constant package, what do you think?

**Review by kristojorg — COMMENTED** [review]
Yep just the mailto but otherwise ready to go!

**Comment by oliviawongnyc** [comment]
@Toxiapo, added DEV_CONTACT_EMAIL to constants package like CONTACT_EMAIL. Also check in ReportAProblem component if the env is 'dev' or 'prod' and switch the email accordingly.

**Review by Toxiapo — APPROVED** [review]
Looks Good!

**Pull request merged**